core/drivers/sam/at91_pmc.h

```diff
@@ -0,0 +1,260 @@
+/* SPDX-License-Identifier: GPL-2.0+ or BSD-3-Clause */
+/*
+ * Copyright (C) 2005 Ivan Kokshaysky
+ * Copyright (C) SAN People
+ *
+ * Power Management Controller (PMC) - System peripherals registers.
+ * Based on AT91RM9200 datasheet revision E.
+ */
+
+#ifndef DRIVERS_CLK_AT91_PMC_H
+#define DRIVERS_CLK_AT91_PMC_H
```

Comment:
This is a local function, `AT91_PMC_H` would be enough.
Otherwise, prefer stick to header file path: `DRIVERS_SAM_AT91_PM_H`

Reply:
Acked